sdks/go/pkg/beam/core/sdf/sdf.go

```diff
@@ -13,42 +13,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package sdf is experimental, incomplete, and not yet meant for general usage.
+// Package sdf contains interfaces used specifically for splittable DoFns.
```

Comment:
Note, even if we submit this now-ish, we can still call it experimental until we have testing and some IO usage to bolster confidence.

Reply:
Done, I prefer to call it experimental at the moment.